Gemfile

```diff
@@ -7,7 +7,7 @@ gemspec
 
 # jquery-rails is used by the dummy application
 gem "jquery-rails"
-gem "mysql2"
+gem "mysql2", "~> 0.3.10"
```

Comment:
None of the tests run, and I couldn't serve it locally. It has something to do with the MySQL adapter that Hubstats is using. I don't think this will affect anything on the current hubstats, but for development it definitely does.